content/docs/concepts/observability/index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Observability
```

Comment:
Since this `Observability` section replaces `Policies and Telemetry`, now there is no `Policies` section in the concepts part of the doc. Don't we want/need a section for that too?

Reply:
Hi Frank,

You're correct, but it seems Policies and Telemetry title was a misnomer and never had a section that properly explained what a policy is at the conceptual level. We have several policy details in the Tasks section, but nothing suitable for concepts. We'll need to spent some time and write a Policies concept section.

I can file an issue to track that we need to create one.

Thoughts?

Reply:
My thoughts on this were as follows:

The "Security" task should be relabeled as "Policy and Security". It already covers policy extension. Separating policy from security will lead to confusion, and overlapping terminology.

Reply:
Do you mean the Security "concepts section" (not task). If so, that sounds reasonable to me.

Reply:
@frankbu sorry, yes. the security concepts doc.

Reply:
I suggest "Security and Policies" because Security is the broad concept and policies is a subset of it and the topic is heavier on security itself, at least currently.

Reply:
I don't have a strong preference, but personally I see it the opposite way. Security is just one of the kind of Policy that Istio can enforce. The fact that the current doc is light on policies is just something that will need to change soon anyway.

Reply:
Also, at a minimum, I think the first couple of paragraphs in the policy/security doc should be changed to have an appropriate intro that corresponds to the changed title.